dashboard/src/components/DetailsPages/SectionError.tsx

```diff
@@ -69,9 +73,10 @@ const SectionError = ({
       <p className="text-2xl font-semibold">
         <FormattedMessage id={message} />
       </p>
-      {errorMessage && errorStatusCode !== HttpStatusCode.Ok && (
+      {((errorMessage && errorStatusCode !== HttpStatusCode.Ok) ||
+        forceErrorMessageUse) && (
         <p>
-          {globalErrorMessage[variant].label}: {errorMessage}
+          {globalErrorMessage[variant].label}: {errorResponseMessage}
```

Comment:
Shouldn't this be something like

```suggestion
          {globalErrorMessage[variant].label}: {errorResponseMessage ?? errorMessage}
```

in the case `errorMessage` is sent as prop to this component?